ani-cli

```diff
@@ -106,15 +120,38 @@ dep_ch() {
 
 # extract the video links from reponse of embed urls, extract mp4 links form m3u8 lists
 get_links() {
-    episode_link="$(curl -e "https://${allanime_base}" -s --cipher "AES256-SHA256" "https://allanimenews.com$*" -A "$agent" | sed 's|},{|\n|g' | sed -nE 's|.*link":"([^"]*)".*"resolutionStr":"([^"]*)".*|\2 >\1|p;s|.*hls","url":"([^"]*)".*"hardsub_lang":"en-US".*|\1|p')"
+    episode_link=$(
+        curl \
+            -e "https://${allanime_base}" \
+            -s \
+            --cipher "AES256-SHA256" "https://allanimenews.com$*" \
+            -A "$agent" |
+            sed 's|},{|\n|g' |
+            sed -nE 's|.*link":"([^"]*)".*"resolutionStr":"([^"]*)".*|\2 >\1|p;s|.*hls","url":"([^"]*)".*"hardsub_lang":"en-US".*|\1|p'
+    )
     case "$episode_link" in
         *crunchyroll*)
-            curl -e "https://${allanime_base}" -s --cipher "AES256-SHA256" "$episode_link" -A "$agent" | sed 's|^#.*x||g; s|,.*|p|g; /^#/d; $!N; s|\n| >|' | sort -nr
+            curl \
+                -e "https://${allanime_base}" \
+                -s \
+                --cipher "AES256-SHA256" "$episode_link" \
+                -A "$agent" |
+                sed 's|^#.*x||g; s|,.*|p|g; /^#/d; $!N; s|\n| >|' |
+                sort -nr
             ;;
         *repackager.wixmp.com*)
```

Comment:
Not sure if the line splitting in this branch is even necessary at all

Reply:
I believe it makes the `sed` (the scary part) stand out. I was even thinking about splitting that into multiple `sed`s for better readability.

Not everyone is as good with sed. Plus, since a lot of patches seem to include changes to the sed, it'll improve the readability of  the changes.

The line (crunchyroll) used to be 153 characters, too much for a curl and a sed if you ask me.